<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="meta_placeholder" native="true"/>
   </item>
   <item>
    <widget class="QTableWidget" name="pv_table">
     <column>
      <property name="text">
       <string>PV name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>title</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>description</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="child_table"/>
   </item>
   <item>
    <widget class="QTextEdit" name="repr_text_edit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
